libraries/rush-lib/src/cli/RushPnpmCommandLineParser.ts

```diff
@@ -477,14 +477,14 @@ export class RushPnpmCommandLineParser {
           }
 
           // Update patchedDependencies to pnpm configuration file
-          this._rushConfiguration.pnpmOptions.updateGlobalPatchedDependencies(newGlobalPatchedDependencies);
+          this._subspace.getPnpmOptions()?.updateGlobalPatchedDependencies(newGlobalPatchedDependencies);
```

Comment:
Should this be an error case if it's undefined and we're trying to update it? Seems like it should be, or we should create the file from the `rush init` template.

Reply:
Updated the PR to handle the pnpm-config.json not exists case